src/app/modules/shared_models/message_model.nim

```diff
@@ -352,6 +352,26 @@ QtObject:
 
     return self.items[ind]
 
+  proc itemSent*(self: Model, messageId: string) =
+    let ind = self.findIndexForMessageId(messageId)
+    if(ind == -1):
+      return
+
+    self.items[ind].outgoingStatus = "send"
```

Comment:
@faust4exe if "send" and "delivered" strings come from the status go, identify them as constants in message_dto.nim file and use constants around.

Reply:
Ok - will do